documentation/modules/proc-installing-crds.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// assembly-evaluation.adoc
+
+[id='proc-install-crds-{context}']
+= Installing {ProductName}
+
+Install {ProductName} with the Custom Resource Definitions (CRDs) required for deployment.
+
+.Prerequisites
+
+* Installation requires a Kubernetes account with cluster admin credentials.
+
+.Procedure
+
+. Log in to the Kubernetes cluster with cluster admin privileges.
+
+. Modify the installation files according to the `kafka` namespace where you will install the {ProductName} Kafka Cluster Operator.
++
+NOTE: By default, the files work in the `myproject` namespace.
++
+* On Linux, use:
++
+
+[source, shell, subs=+quotes]
+----
+sed -i 's/namespace: .\*/namespace: kafka/' install/cluster-operator/*RoleBinding*.yaml
+----
++
+* On Mac, use:
++
+[source, shell, subs=+quotes]
+----
+sed -i '' 's/namespace: .\*/namespace: kafka/' install/cluster-operator/*RoleBinding*.yaml
+----
+. Deploy the CRDs and role-based access control (RBAC) resources to manage the CRDs.
++
+[source, shell, subs=+quotes ]
+----
+kubectl create ns kafka
```

Comment:
It weird and confusing to change the namespace in the resources to a namespace that you're not created yet (if it hasn't been explained that you'll be creating a namespace).

Reply:
Let's try:

. Modify the installation files to reference a _new_ `kafka` namespace where you will install the {ProductName} Kafka Cluster Operator.
. Create the namespace.
. Deploy the CRDs and role-based access control (RBAC) resources to manage the CRDs.